### PR TITLE
Grammar corrections

### PR DESCRIPTION
**Pull request by brouberol**

### CODE REVIEW

**Comment by VincentCasse** [comment]
LGTM

File: en/_posts/2016-04-20-marathon-health-checks.md [hunk 1]
```diff
@@ -6,23 +6,23 @@ author: devatoria
 lang: en
 ---
 
-Health checks are useful to ensure that your container is running; and better, is running as expected. You can add multiple health checks per application. A health check can be of two types:
+Health checks are useful to ensure that your containers are running; and even better, are running as expected. You can add multiple health checks per application. A health check can be of two types:
```
Comment (yadutaf) [inline en/_posts/2016-04-20-marathon-health-checks.md]: - (non blocking suggestion): are behaving as expected
- (non blocking suggestion): Marathon supports both high level HTTP and lower level TCP health checks

File: en/_posts/2016-04-20-marathon-health-checks.md [hunk 2]
```diff
@@ -6,23 +6,23 @@ author: devatoria
...
 
 - HTTP
 - TCP
 
-The only difference is that the HTTP check is checking a path (using http://localhost:port/path), and is able to ignore some HTTP status code (informational ones, from 100 to 199).
+The only difference is that the HTTP check is given a path (using http://localhost:port/path), and is able to ignore some HTTP status codes (informational ones, from 100 to 199).
```
Comment (yadutaf) [inline en/_posts/2016-04-20-marathon-health-checks.md]: HTTP checks allow for better, higher level checks on a specific URL like ``http://localhost:port/my/health/path``. It automatically ignores informational status codes in the range 100 to 199.

File: en/_posts/2016-04-20-marathon-health-checks.md [hunk 3]
```diff
@@ -6,23 +6,23 @@ author: devatoria
...
 
-#Configure a health check for a nginx application
+# Configure a health check for a nginx application
 
-In the previous tutorial, we ran a simple nginx application. Let's add some health check for this application.
+In the [previous tutorial](https://community.runabove.com/kb/en/docker/quick-start-with-marathon.html), we ran a simple nginx container. Let's now configure some HTTP health check for this application.
```
Comment (yadutaf) [inline en/_posts/2016-04-20-marathon-health-checks.md]: :+1: for the link. Pro-tip: you can use relative links

File: en/_posts/2016-04-20-marathon-health-checks.md [hunk 4]
```diff
@@ -6,23 +6,23 @@ author: devatoria
...
 
 ![Main interface](/kb/images/2016-04-20-marathon-health-checks/health_check.png)
 
-- **Grace Period**: the time to wait from the launch of the task where health check errors will be ignored *(useful if you have a task which is long to be healthy and do not want your health check to fail because of this)*
-- **Interval**: the time to wait before each health check
-- **Timeout**: the time to wait before a health check is considered as failed regardless of the result
+- **Grace Period**: health check failures are ignored within this number of seconds or until the task becomes healthy for the first time *(useful if you have a task with a long startup time, to avoid having the  health check failing because of this)*.
```
Comment (yadutaf) [inline en/_posts/2016-04-20-marathon-health-checks.md]: reader question: is the container in the load balancer for the initial grace period or is unrelated ? Use-case: I'd want my container to be registered in the load balancer only when it is ready

File: en/_posts/2016-04-20-marathon-health-checks.md [hunk 5]
```diff
@@ -6,23 +6,23 @@ author: devatoria
...
+- **Interval**: number of secondsto wait before each health check
```
Comment (yadutaf) [inline en/_posts/2016-04-20-marathon-health-checks.md]: - seconds to wait (space type)
- s/before/between/

File: en/_posts/2016-04-20-quick-start-with-marathon.md [hunk 6]
```diff
@@ -6,16 +6,16 @@ author: devatoria
 lang: en
 ---
 
-Welcome to the quick start tutorial of the PaaS Docker lab. This document will guide you to launch your first application and play with it.
+Welcome to the quick start tutorial of the PaaS Docker lab. This document will help you launch your first application using the Marathon Web UI.
```
Comment (yadutaf) [inline en/_posts/2016-04-20-quick-start-with-marathon.md]: Unless "PaaS Docker" is an official name, I'd recommend the "Docker PaaS" formulation to sound less frenchy ;)

File: en/_posts/2016-04-20-quick-start-with-marathon.md [hunk 7]
```diff
@@ -6,16 +6,16 @@ author: devatoria
...
 
-#Access to your interface
+#Access to your Web UI
```
Comment (yadutaf) [inline en/_posts/2016-04-20-quick-start-with-marathon.md]: suggestion: make the "Marathon" explicit. That will improve readability for people "scanning though" your tutorial. For example "Access your Marathon Web UI"

**Comment by yadutaf** [comment]
I may have missed it as I'm only reading the diff and went too quickly through the initial version, but I think, as a reader, I'd value a quick introduction on Mesos/Marathon concepts, especially, We hear about "Mesos Slave" from the first paragraph, but what it is referring to is not obvious from the guide. This is not blocking by any mean, but I think it's worth considering it as an improvement in a future PR for users who would discover your product from the documentation.

File: en/_posts/2016-04-20-quick-start-with-marathon.md [hunk 8]
```diff
@@ -6,16 +6,16 @@ author: devatoria
...
 
-After requesting your stack deployment through RunAbove labs section, you can access your Marathon web access using the given URL.
+Once your Mesos slave has been installed, you'll receive an email containing the URL and credentials of your marathon.
 
 
 ![Main interface](/kb/images/2016-04-20-quick-start-with-marathon/marathon.png)
- 
-Here you can see a lot of things you can do. Let's look around:
+
+As you can see, there's a lof of things you can do. Let's have a look:
```
Comment (yadutaf) [inline en/_posts/2016-04-20-quick-start-with-marathon.md]: (non blocking suggestion) "This screenshot gives a first taste of what can be done with Marathon:"

File: en/_posts/2016-04-21-marathon-container-metrics.md [hunk 9]
```diff
@@ -6,17 +6,17 @@ author: devatoria
 lang: en
 ---
 
-With the [PaaS Timeseries lab](https://www.runabove.com/iot-paas-timeseries.xml), you are able to push metrics about your containers and make some graphs using Grafana, such as:
+We integrated the PaaS Docker ith the [PaaS Timeseries lab](https://www.runabove.com/iot-paas-timeseries.xml), to enable you to graph your application metrics about your containers in a Grafana dashboard. We send metrics about:
```
Comment (yadutaf) [inline en/_posts/2016-04-21-marathon-container-metrics.md]: s/ith/with/ (missing 'w')

Comment (yadutaf) [inline en/_posts/2016-04-21-marathon-container-metrics.md]: to allow you to...

Comment (yadutaf) [inline en/_posts/2016-04-21-marathon-container-metrics.md]: we automatically send metrics for

File: en/_posts/2016-04-21-marathon-load-balancer.md [hunk 10]
```diff
@@ -6,13 +6,18 @@ author: devatoria
 lang: en
 ---
 
-Marathon load balancer can be configured to use a specific vhost, so the URL to access your application will be more user-friendly. For example, let's say we have a nginx application and we want it to be available at `http://myapp.info`.
+The Marathon load balancer can be configured to use a specific VHost (virtual host), to make you application URL more user-friendly. For example, let's say we have a nginx application and we want it to be available at `https://myapp.info`.
```
Comment (yadutaf) [inline en/_posts/2016-04-21-marathon-load-balancer.md]: suggestion: replace vhost notion by "domain name". That's easier to understand without prior LB or Apache knowledge.

File: en/_posts/2016-04-21-marathon-load-balancer.md [hunk 11]
```diff
@@ -6,13 +6,18 @@ author: devatoria
...
-Considering your domain name is pointing to your load balancer URL (http://username.lb.sbg-1.containers.ovh.net), you just have to add the following labels:
+Considering your domain name is pointing to your load balancer URL (https://<username>.lb.sbg-1.containers.ovh.net), you just have to add the following labels:
 
 - **HAPROXY\_0\_MODE**: "http"
 - **HAPROXY\_0\_VHOST**: "myapp.info"
 
 ![Main interface](/kb/images/2016-04-21-marathon-load-balancer/haproxy.png)
 
-If you want more information about how to configure your HAProxy load balancer, please [read this documentation](https://github.com/mesosphere/marathon-lb#haproxy-configuration).
+If you want more information about how to configure your HAProxy load balancer, please read the [marathon-lb documentation](https://github.com/mesosphere/marathon-lb#haproxy-configuration). You'll see for example how to set weight on backends, how to enable sticky sessions on each backend.
```
Comment (yadutaf) [inline en/_posts/2016-04-21-marathon-load-balancer.md]: For more information about...

File: en/_posts/2016-04-21-marathon-load-balancer.md [hunk 12]
```diff
@@ -6,13 +6,18 @@ author: devatoria
...
+
+## Configuring ACLs 
+
+**Note**: there is an undocumented (for the moment) feature in marathon-lb: you can configure ACLs (Access Control List) on your applications, thus only allowing given IPs or networks to have access to your containers.
```
Comment (yadutaf) [inline en/_posts/2016-04-21-marathon-load-balancer.md]: Seems weird to speak of undocumented feature in... documentation ;) Do you mean upstream documentation ?

File: en/_posts/2016-04-22-introduction-to-docker-with-mesos-marathon.md [hunk 13]
```diff
@@ -8,43 +8,44 @@ lang: en
 
 #Welcome to the Docker PaaS lab!
 
-This article is here to guide your first steps through our lab, with a little explanation of what is Docker, Apache Mesos and Marathon, and how you will be able to use it using our technology stack.
+In this series of articles, we'll guide you through using our Docker PaaS, explaining what [Docker](https://www.docker.com/), [Apache Mesos](https://mesos.apache.org/) and [Marathon](https://mesosphere.github.io/marathon/) are, and how we use them to provide you with a managed Docker as a Service platform.
 
 ##What is Docker?
 
-Docker is now a standard containerization solution. It uses multiple technologies, like **control groups** and **namespaces**, to isolate your application.
+First, you may be wondering what Docker is. It that's the case, we'll try to sum it up in a couple of words.
```
Comment (yadutaf) [inline en/_posts/2016-04-22-introduction-to-docker-with-mesos-marathon.md]: Suggestion: I'd remove this line. It does not bring information.

File: en/_posts/2016-04-22-introduction-to-docker-with-mesos-marathon.md [hunk 14]
```diff
@@ -8,43 +8,44 @@ lang: en
...
-The Docker PaaS lab is based on all these technologies, adding some other cool stuff. Our lab is stack managed, it means that **we** deploy and manage your infrastructure.
+The Docker PaaS lab is based on all these software stacks, adding some other cool stuff, like redirecting your application Logs to our PaaS Logs, and your application metrics our PaaS Timeseries.. Our lab is stack managed, it means that **we** deploy and manage your infrastructure.
 
 ##Your resources are dedicated
 
-We are not providing a pool of slaves where everybody launches containers. You have dedicated slaves from OVH Cloud, and dedicated resources. Your containers are running on your slaves, and only yours, without noisy neighbours.
+We are not providing a pool of slaves where everybody launches containers. You have **dedicated** Public Cloud instances, and dedicated resources. Your containers are running on your instances, and only yours, without noisy neighbours.
+During the beta, you'll be limited to a single (and free) [VPS SSD 3](https://www.ovh.com/fr/vps/vps-ssd.xml). However, you'll be able to add more resources as we advance, not limited to the OVH cloud.
 
 ##We manage everything, but...
```
Comment (yadutaf) [inline en/_posts/2016-04-22-introduction-to-docker-with-mesos-marathon.md]: Suggestion: As a reader, when reading "but.." when quickly scanning, I understand "we don't manage all, but we say so". Which is not true :)

File: en/_posts/2016-04-22-introduction-to-docker-with-mesos-marathon.md [hunk 15]
```diff
@@ -8,43 +8,44 @@ lang: en
...
+We manage Mesos masters, ZooKeepers, load balancers and Mesos slaves. However, we provide you with an access to the marathon web UI and to your HAProxy load balancer stats.
 
-##We do not only provide you containers...
+##We do not only give you containers
+We provide you with a full stack, integrated with the OVH ecosystem. It means that your stack is fully extensible: as the beta evolves, you'll be able to add new instances, as your needs evolve, from any kind (Public Cloud, Dedicated Server, etc).
 
-We provide you a full stack with an OVH environment integration. When you are asking for a new stack, we deploy you OVH Cloud instances and provision it to integrate it to the cluster. It means that you stack is fully extensible. You can add new instances as you want, and to fill your needs at the moment.
+To easily assess if you have enough resources for your running applications, we provide you with [a way to have concrete metrics using the PaaS Timeseries lab](/kb/en/docker/marathon-container-metrics.html), and exploit them with Grafana dashboards.
```
Comment (yadutaf) [inline en/_posts/2016-04-22-introduction-to-docker-with-mesos-marathon.md]: concrete first meaning is "béton" in english ^^

**Comment by yadutaf** [comment]
Nice improvement Thanks! I've taken the occasion to point to a couple of additional improvement areas. Aside from some typos, nothing blocking to me.

File: en/_posts/2016-04-20-marathon-health-checks.md (hunk 4 above)
Comment (brouberol) [inline en/_posts/2016-04-20-marathon-health-checks.md]: I think it is, but I'd have to check.

**Comment by brouberol** [comment]
@VincentCasse Signed :)

**Comment by brouberol** [comment]
@yadutaf Thanks so much for the thorough review. I've added a link from the quickstart to the introduction, as a "first and foremost" section.

**Comment by yadutaf** [comment]
LGTM. @VincentCasse I'll let you merge if that's good for you too.

**Comment by brouberol** [comment]
Up?

**Comment by yadutaf** [comment]
Merged

**Comment by brouberol** [comment]
Great, thanks!
 
--
B
 
 
On Mon, May 23, 2016, at 01:42 PM, Jean-Tiare Le Bigot wrote:
> Merged
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub[1]
>

Links:

  1. https://github.com/runabove/knowledge-base/pull/125#issuecomment-220957535

**Pull request merged**